src/main/java/seedu/address/model/timingrecommender/TimingRecommender.java

```diff
@@ -109,6 +107,7 @@ public List<TimePeriod> getAllTimings() {
             List<HourBlock> availableHourBlocks = TimeUtil.getFreeCommonIntervals(day, schedules);
             periods.addAll(TimeUtil.mergeTimeSlots(availableHourBlocks));
         }
+        logger.info(String.format("%d possible timings", periods.size()));
```

Comment:
Good use of Defensive Programming Techniques!

Reply:
Thanks! The code was being difficult.